<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>105</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(97, 120, 202);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditport">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>1883</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>40</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(97, 120, 202);</string>
    </property>
    <property name="text">
     <string>提示:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>54</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>device0&amp;a1uEFOvKWyp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>56</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>账号:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>80</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>密码:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>d966bd0269b388f4fd161fa625d8f8fea934f0ebe7a9855b889611ff86826cbc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonConnect">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>10</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHost">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>6</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>a1uEFOvKWyp.iot-as-mqtt.cn-shanghai.aliyuncs.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>31</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTopic">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>135</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>/sys/a1uEFOvKWyp/device0/thing/event/property/post</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>136</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>Topic:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPublish">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>160</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>发布</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSubscribe">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>订阅</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>160</y>
      <width>64</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>Message:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMessage">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>160</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>This is a test message</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="receive_msg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;宋体&quot;;</string>
    </property>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
